Dockerfile

```
@@ -3,7 +3,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 ############# golang-base
-FROM golang:1.16.15 AS golang-base
+FROM golang:1.19.3 AS golang-base
```

Let's use the latest 1.19 version
```suggestion
FROM golang:1.19.5 AS golang-base
```